app/controllers/books/edit.js

```diff
@@ -0,0 +1,72 @@
+import EditController from 'alpha-amber/controllers/application/edit';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class BookEditController extends EditController {
+  @service fetch
+
+  successTransitionTarget = 'books.show'
+  @tracked showScanner = false;
+  @tracked devices = [];
+  @tracked device = null;
+  @tracked lookingUpIsbn = false;
+  @tracked lookupIsbnError = false;
+
+  @action
+  coverPhotoLoaded(file) {
+    const book = this.model;
+    book.set('coverPhoto', file.data);
+  }
+
+  @action
+  submit() {
+    if (!this.model.isbn) {
+      this.model.isbn = null;
+    }
+
+    this.model.save().then(() => {
```

Comment:
I think if you do `super.submit()` here the EditController's submit method will do exactly the same stuff